managed/ui/src/components/config/Storage/ConfigType.js

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) YugaByte, Inc.
+//
+// Author: Nishant Sharma(nishant.sharma@hashedin.com)
+//
+// This file will help us to maintain the different type of configs
+// along with their respective input field component except for AWS.
+
+import { YBTextInputWithLabel } from "../../common/forms/fields";
+
+const YBTextInputWithLabelComponent = YBTextInputWithLabel;
```

Comment:
not sure I get this, why not use `YBTextInputWithLabel` directly?

Reply:
Silly mistake.